api/tests/test_check_dem.py

```diff
@@ -0,0 +1,42 @@
+from shapely.geometry import Polygon
+
+from hyp3_api.validation import check_intersect, get_coverage_shapes_from_geojson
+
+
+def test_dem_validation_intersections():
```

Comment:
testing `check_dem_coverage()` would be more appropriate, rather than calling the building-block  `get_coverage_shapes_from_geojson()` and `check_intersect()` functions